src/main/java/step2/ui/ResultView.java

```diff
@@ -0,0 +1,47 @@
+package step2.ui;
+
+import step2.domain.Ladder;
+import step2.domain.Line;
+import step2.domain.Name;
+import step2.domain.Point;
+
+import java.util.List;
+
+public class ResultView {
+
+    public static final int PARTICIPANT_MATCH_LADDER_SIZE = 6;
+    public static final String EXECUTION_RESULT = "실행 결과";
+    public static void printResultMessage() {
+        System.out.println(EXECUTION_RESULT);
+    }
```

Comment:
변수와 메서드 사이(메서드와 메서드 사이)에는 개행을 추가하는게 가독성에 도움이 되는 것 같아요 😃 
```suggestion
    public static final int PARTICIPANT_MATCH_LADDER_SIZE = 6;
    public static final String EXECUTION_RESULT = "실행 결과";
    
    public static void printResultMessage() {
        System.out.println(EXECUTION_RESULT);
    }
```